spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/SparkTypeConverter.java

```diff
@@ -43,6 +79,33 @@ public static DataType toSparkType(Type gravitinoType) {
       return DataTypes.FloatType;
     } else if (gravitinoType instanceof Types.DoubleType) {
       return DataTypes.DoubleType;
+    } else if (gravitinoType instanceof Types.ByteType) {
+      return DataTypes.ByteType;
+    } else if (gravitinoType instanceof Types.ShortType) {
+      return DataTypes.ShortType;
+    } else if (gravitinoType instanceof Types.LongType) {
+      return DataTypes.LongType;
+    } else if (gravitinoType instanceof Types.DecimalType) {
+      Types.DecimalType decimalType = (Types.DecimalType) gravitinoType;
+      return DataTypes.createDecimalType(decimalType.precision(), decimalType.scale());
+    } else if (gravitinoType instanceof Types.BinaryType) {
+      return DataTypes.BinaryType;
+    } else if (gravitinoType instanceof Types.FixedCharType) {
+      Types.FixedCharType charType = (Types.FixedCharType) gravitinoType;
+      return CharType.apply((charType.length()));
+    } else if (gravitinoType instanceof Types.VarCharType) {
+      Types.VarCharType varCharType = (Types.VarCharType) gravitinoType;
+      return VarcharType.apply(varCharType.length());
+    } else if (gravitinoType instanceof Types.DateType) {
+      return DataTypes.DateType;
+    } else if (gravitinoType instanceof Types.TimestampType
```

Comment:
@mchades please help to reivew , especially for timestamp related types

Reply:
This part LGTM

Reply:
thanks